packages/sdk/typescript/human-protocol-sdk/src/escrow.ts

```diff
@@ -949,7 +1059,13 @@ export class EscrowClient extends BaseEthersClient {
     try {
       const escrowContract = this.getEscrowContract(escrowAddress);
 
-      return escrowContract.getBalance();
+      try {
+        return await escrowContract.remainingFunds();
```

Comment:
We should probably have 2 functions:
- getTotalBalance
- getRemainingFunds

Reply:
What's the difference between these two? I think we just need balance information. All the extra funds are just available for withdraw.